lib/grape/error_formatter/json.rb

```diff
@@ -3,14 +3,23 @@ module ErrorFormatter
     module Json
       class << self
         def call(message, backtrace, options = {}, env = nil)
-          message = Grape::ErrorFormatter::Base.present(message, env)
+          result = wrap_message(Grape::ErrorFormatter::Base.present(message, env))
 
-          result = message.is_a?(String) ? { error: message } : message
           if (options[:rescue_options] || {})[:backtrace] && backtrace && !backtrace.empty?
             result = result.merge(backtrace: backtrace)
           end
           MultiJson.dump(result)
         end
+
+        private
+
+        def wrap_message(message)
+          if message.is_a?(Exceptions::ValidationErrors)
+            return message
```

Comment:
Remove `return`, that's implied.  Expand the condition to be consistent:

```ruby
if message.is_a?(...)

elsif message.respond_to?(:to_h)

else

end
```

Or better

```
case
when ...
else
...

I am not in love with the check for `to_h`, I think you want to check `is_a?(Hash)` - or are you seeing a Mash or something? In which case I think it should still be checked explicitly.